DayrSuryan/DSEthiop7.xml

```diff
@@ -5,7 +5,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title>Maṣḥafa saʿātāt and other texts</title>
+            <title>Book of the Hours and related texts</title>
```

Comment:
I would say: "Book of Hours, Nicene Creed, Sebhata fequr poems, and other texts.